src/nixmultitag.cc

```diff
@@ -36,6 +36,16 @@ namespace nixmultitag {
         currObj.addSource(input.str(2));
     }
 
+    void create_feature(const extractor &input, infusor &output)
+    {
+        nix::MultiTag currObj = input.entity<nix::MultiTag>(1);
+
+        nix::LinkType link_type = nixgen::get_link_type(input.num<uint8_t>(3));
```

Comment:
Similar to 

nix::DataType extractor::dtype(size_t pos) const 

(in arguments.h) I'd propose to keep this conversion in extractor, such that basically is located all in one place. Just move it from nixgen to extractor.